ando/tools/generator/AnDOGenerator.py

```diff
@@ -238,6 +239,31 @@ def generate_metadata_files(self):
                 if re.compile(regex).match(mfile.name):
                     create_file(mfile, parents[(3-level)] / mfile.name,
                                 mode='copy')
+        # todo : rename methods 'generate_metadata_files' -> 'organize_metadata_files'
+        # add new method 'generate_metadata_files' that is calling all 'create_metadata_file_*' methods
+
+    def create_metadata_file_participants(self) -> bool:
```

Comment:
These methods also need an good docstring as users are supposed to overwrite them.